Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,39 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    ps = formula.prefix
+    a = "#{ps}/bin"
+    b = "#{ps}/lib"
+    c = "#{ps}/libexec"
+    d = "#{ps}/sbin"
+    e = "#{ps}/share"
+
+    # THIS DOESN'T WORK YET. ASK MIKE/JACK/TIM WHAT I'M DOING WRONG.
+    if text =~ /bash_completion/
```

Comment:
@mikemcquaid @jacknagel @tdsmith Any idea why this chunk of code isn't working? I'm doing something wrong but I can't put my finger on what yet.